### PR TITLE
Fastluke2

### PR DESCRIPTION
**Pull request by Luke1298**

### CODE REVIEW

File: core/app.js [hunk 1]
```diff
@@ -96,6 +102,13 @@ app.use(_.get("/message", function *() {
   yield this.render('msg', {layout: this.USE_LAYOUT, all_messages: all_messages, all_tasks: all_tasks});
 }));
 
+app.use(_.get("/calander-request", function*(){
+  var client = db();
+  var allCalendarEvents = yield client.query("SELECT * FROM calendar;");
+  var isAdmin = false;
+  yield this.render('calendar', {layout: "simple_layout", date: new Date(), allCalendarEvents: allCalendarEvents, user: this.session.user, isAdmin:isAdmin});
```
Comment (Luke1298) [inline core/app.js]: I have a better Idea!

**Comment by Luke1298** [comment]
@webnard tests on this are failing. Help?

File: core/templates/calendar.html [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
+
 <script>
-  <% var roomsHLRC =  ["Recording Studio", "1141A", "1141B", "1141C", "1141D"] %> <!--Should these be passed in from app.js?-->
+  <% var roomsHLRC =  ["Recording Studio", "1141A", "1141B", "1141C", "1141D"] %> //Should these be passed in from app.js?-->
   <% var roomsHTRSC = ["B160", "B164", "B181", "B013", "B161", "1161", "B003", "1131A", "1133A", "1133B"] %>
   window.rooms = {};
   window.rooms.HLRC = ("<%- (roomsHLRC.toString()) %>").split(",");
   window.rooms.HTRSC = ("<%- roomsHTRSC.toString() %>").split(",");
 </script>
+<script src="http://code.jquery.com/jquery-2.1.4.min.js">//VERY HACKY, but it will have to do for now. Some thrid party is preventing JQuery from loading all the way. Now Jquery is "double added" this is in not good practice. #HammerTime</script>
```
Comment (webnard) [inline core/templates/calendar.html]: you can delete this line here

**Comment by Luke1298** [comment]
@webnard These tests are failing :(

**Comment by webnard** [comment]
@Luke1298 Not 100% sure what's going on at the moment, but when I request a room as a non-admin I get this error in my Node.js terminal:

```Javascript
{ [Error: Invalid login]
  code: 'EAUTH',
  response: '503 5.5.1 Error: authentication not enabled',
  responseCode: 503 }
```

**Comment by Luke1298** [comment]
:(

**Comment by Luke1298** [comment]
Okay I am in the middle of fastluke3, do you have time to investigate this rn?

**Comment by webnard** [comment]
@Luke1298 can you rebase this one first?

**Comment by Luke1298** [comment]
Re based.

**Comment by Luke1298** [comment]
@webnard

**Comment by Luke1298** [comment]
@webnard Still failed :(

**Comment by webnard** [comment]
@Luke1298 Try running npm test on different commits in the history until you find the one where it stops breaking, then we can try to see what happened.

Run `git log`, copy a hash, and run `git checkout <hash-number>`. `npm i && npm t` and try over and over. I'm off to bed, but I'll be happy to look at this in the morning

**Comment by Luke1298** [comment]
@webnard Okay will do.

**Comment by Luke1298** [comment]
Looks like: e89af2a103093e952986ff22ad42dedeaeecba88
is the culprit. I think it has something to do with querying...

File: core/templates/simple_layout.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
```
Comment (trevren11) [inline core/templates/simple_layout.html]: Is this file a little redundant? Could we just remove some of the options on the regular layout?

**Comment by Luke1298** [comment]
:+1:

File: core/templates/calendar.html [hunk 4]
```diff
@@ -1,8 +1,7 @@
 <link rel="stylesheet" type="text/css" href="/css/reset.css">
 <link rel="stylesheet" type="text/css" href="/css/calendar.css">
-
 <script>
-  <% var roomsHLRC =  ["Recording Studio", "1141A", "1141B", "1141C", "1141D"] %> //Should these be passed in from app.js?-->
+  <% var roomsHLRC =  ["Recording Studio", "1141A", "1141B", "1141C", "1141D"] %> <!--Should these be passed in from app.js?-->
```
Comment (webnard) [inline core/templates/calendar.html]: wait a second!

File: core/templates/simple_layout.html (hunk 3 above)
Comment (webnard) [inline core/templates/simple_layout.html]: Agreed--let's see how different we actually need these, and then we might consider merging them into one file in the future.

**Pull request merged**